tests/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -87,6 +87,17 @@ TEST (nnstreamer_capi_construct_destruct, failure_02_n)
   EXPECT_EQ (status, ML_ERROR_STREAMS_PIPE);
 }
 
+#define wait_for_start(handle, state, status) \
+do {\
+  int counter = 0;\
+  while (state == ML_PIPELINE_STATE_PAUSED && counter < 20) {\
```

Comment:
Question: why the pipeline-state is 'PAUSED', not 'PLAYING'? (Now ml_pipeline_construct() makes the pipeline paused.)

Reply:
The sequence is:
```
1: construct()
2: state == PAUSED
3: play()
4: state == PAUSED
5: some time after play, the pipeline starts asynchronously...
6: state == PLAYING
```

The time required for step 5 varies and it causes QEMU-ARM systems to use more than 200ms sometimes.